tests/server_test_cases/config_tests.py

```diff
@@ -440,7 +440,7 @@ def test_analytics_id(self):
         config.set(analytics_id='analytics_id_xyz')
 
         doc = self.go('/haiti/create')
-        assert "getTracker('analytics_id_xyz')" in doc.content
+        assert "gaProperty': 'analytics_id_xyz'" in doc.content
```

Comment:
Thanks for reviewing again!
This code does work as long as analytics_id is configured because Analytics tag was already set up in tagmanager.google.com. I would modify the assert string also.